packages/common/src/ssh/base-client.ts

```diff
@@ -5,6 +5,11 @@ import { tryParseJson } from '../json'
 import { Logger } from '../log'
 import { formatPublicKey, parseKey } from './keys'
 
+export interface BaseUrl {
```

Comment:
Not a huge difference, but I would use a URL type which is more extendable. what if there's a base path, e.g, https://livecycle.run/myorg?